test/IDE_tests.jl

```diff
@@ -204,3 +204,56 @@ w_real  = [1/x^2 for x in xs]
 # plot!(xs,u_predict)
 # plot(xs,w_real)
 # plot!(xs,w_predict)
+
+println("Improper (0, Inf) Integral Test")
+
+@parameters x
+@variables u(..)
+I = Integral(x in ClosedInterval(0, Inf))
+eqs = I(u(x)) ~ π/(3*sqrt(3))
```

Comment:
here is the same problem as before. The system has many solutions, therefore optimization strives for one of them. exp (x) / (exp (2 * x) + 3) one of the many solutions.
First of all, the problem is not with infinity here, try with the finite domain it will be the same.

it like trying to solve:
sum(u(x)) = 1 # x is some set
u(1)=1 

result for this test.
![image](https://user-images.githubusercontent.com/12683885/145562033-0356e41e-8dde-4890-9cde-d9b02ba0fe27.png)

Reply:
I will add tests